pb_plugins/dcsdkgen/struct.py

```diff
@@ -11,8 +11,13 @@ class Struct(object):
     """ Struct """
 
     def __init__(self, package, template_env, pb_struct):
+        try:
+            self._template = template_env.get_template("struct.j2")
+        except:
```

Comment:
We should __NEVER__ do that! We might miss some exceptions.

Reply:
Right, sorry :sweat_smile:. Is it better like this? Or should I avoid the `try...except` syntax altogether?